From 78009fcbeebe959e554b6f43d4d0797c65fa99f1 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:28:58 -0400
Subject: [alsa] hda: Enable beep for ASUS 1015E

Message-id: <140972823573-247-patchset-jkysela@redhat.com>
Patchwork-id: 90947
O-Subject: [RHEL 7.1 PATCH 246/442] ALSA: hda - Enable beep for ASUS 1015E
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The `lspci -nnvv` output contains (wrapped for line length):

00:1b.0 Audio device [0403]:
Intel Corporation 7 Series/C210 Series Chipset Family
High Definition Audio Controller [8086:1e20] (rev 04)
Subsystem: ASUSTeK Computer Inc. Device [1043:115d]

Signed-off-by: W. Trevor King <wking@tremily.us>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: a4b7f21d7b42b33609df3f86992a8deff80abfaf)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index eb5eebe..ce564e1 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -994,6 +994,7 @@ static int alc_codec_rename_from_preset(struct hda_codec *codec)
 
 static const struct snd_pci_quirk beep_white_list[] = {
  SND_PCI_QUIRK(0x1043, 0x103c, "ASUS", 1),
+ SND_PCI_QUIRK(0x1043, 0x115d, "ASUS", 1),
  SND_PCI_QUIRK(0x1043, 0x829f, "ASUS", 1),
  SND_PCI_QUIRK(0x1043, 0x8376, "EeePC", 1),
  SND_PCI_QUIRK(0x1043, 0x83ce, "EeePC", 1),
-- 
1.7.1